components/productImage/leftarrow/leftarrow.js

```diff
@@ -0,0 +1,18 @@
+import  { Component } from "react";
```

Comment:
Rename folder productImage . Keep it in product . Avoid Camelcasing names in folders